airsonic-main/src/main/webapp/WEB-INF/jsp/playQueue.jsp

```diff
@@ -161,11 +165,11 @@
     }
 
     function initAutoHide() {
-        $(window).mouseleave(function (event) {
+        $(".playlistframe").mouseleave(function (event) {
```

Comment:
This change was necessary as the eventlistener on the window seems to get triggered several times with Firefox. Also changed in #1740 (was necessary there as well).